src/App.js

```diff
@@ -37,9 +37,12 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray[0]}/>
```

Comment:
same here what if we have 5000 objects in that tweet array this is no dinamic but you can make it with map 

```
tweetsArray.map(item => ...)
```